lib/backend/k8s/conversion/conversion.go

```diff
@@ -192,9 +192,9 @@ func (c Converter) PodToWorkloadEndpoint(pod *kapiv1.Pod) (*model.KVPair, error)
 				var modelProto numorstring.Protocol
 				switch containerPort.Protocol {
 				case kapiv1.ProtocolUDP:
-					modelProto = numorstring.ProtocolFromString("udp")
+					modelProto = numorstring.ProtocolFromString("UDP")
```

Comment:
Hmm, thought I'd added a comment but perhaps I didn't

I don't believe this package needs updating - as it was used for converting between the v1 API and the v1 backend data